### PR TITLE
Cj landmarks automation

### PR DESCRIPTION
**Pull request by calebsjames**
Needs some CSS fixes but functionality is there

### CODE REVIEW

**Review by SubtleCo — DISMISSED** [review]
and the gods wept at the beauty

**Review by peytonkemp — DISMISSED** [review]

**Review by SubtleCo — DISMISSED** [review]
Love this. Love you.

File: scripts/monaco/landmarks/Landmarks.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const Landmark = (landmark) => {
+    return `
+        <article class="countryContainer__card">
+            <img class="countryContainer__card-img" src="${landmark.image}"</div>
```
Comment (SubtleCo) [inline scripts/monaco/landmarks/Landmarks.js:4]: Close image tag, remove rogue div closer

**Review by peytonkemp — APPROVED** [review]
lovely

**Review by SubtleCo — APPROVED** [review]
👍

**Review by morriscodez — APPROVED** [review]
Make sure you change Landmarks.js

line 4 `````` to replace ```</div>```

**Pull request merged**